yt_dlp/extractor/mxplayer.py

```diff
@@ -125,3 +126,40 @@ def _real_extract(self, url):
             'series': try_get(video_dict, lambda x: x['container']['container']['title']),
             'thumbnails': thumbnails,
         }
+
+
+class MxplayerShowIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?mxplayer\.in/show/(?P<display_id>[-\w]+)-(?P<id>\w+)/?(?:$|[#?])'
+    _TESTS = [{
+        'url': 'https://www.mxplayer.in/show/watch-chakravartin-ashoka-samrat-series-online-a8f44e3cc0814b5601d17772cedf5417',
+        'playlist_mincount': 440,
+        'info_dict': {
+            'id': 'a8f44e3cc0814b5601d17772cedf5417',
+        }
+    }]
+
+    def _entries(self, show_id):
+        show_url = "https://api.mxplay.com/v1/web/detail/tab/tvshowseasons?type=tv_show&id={}&device-density=2&platform=com.mxplay.desktop&content-languages=hi,en".format(
+            show_id)
+        episodes_url = "https://api.mxplay.com/v1/web/detail/tab/tvshowepisodes?type=season&id={}&device-density=1&platform=com.mxplay.desktop&content-languages=hi,en&{}"
+        show_json = self._download_json(show_url, video_id=show_id, headers={'Referer': 'https://mxplayer.in'})
+        page_num = 0
+        for season in show_json.get('items') or []:
+            season_id = try_get(season, lambda x: x['id'], compat_str)
+            next_url = ''
+            while next_url is not None:
+                page_num += 1
+                season_json = self._download_json(episodes_url.format(season_id, next_url),
+                                                  video_id=season_id,
+                                                  headers={'Referer': 'https://mxplayer.in'},
+                                                  note='Downloading JSON metadata page %d' % page_num)
+                next_url = season_json['next']
```

Comment:
```suggestion
                next_url = season_json.get('next')
```

Reply:
Why? `next` is confirmed to exist. It will not exist only when `season_json` is faulty. Therefore it should not proceed and throw and error. If i use `.get` it will return None and no error will be thrown.

Reply:
say, the api changes slightly and `next` no longer exists. Then, we can still download 1 page worth of videos. No point in breaking that by using `['next']`. Generally, any key that is not necessary for the video download itself should be accessed with `get`

Reply:
Got it. Lemme fix.